Formula/abi-compliance-checker.rb

```diff
@@ -3,6 +3,7 @@ class AbiComplianceChecker < Formula
   homepage "http://ispras.linuxbase.org/index.php/ABI_compliance_checker"
   url "https://github.com/lvc/abi-compliance-checker/archive/1.99.9.tar.gz"
   sha256 "52b0daca89fcda73cde126497c8015ca823417074ba02fcff68b7acf2f45e516"
+  revision 1
```

Comment:
It failed in that case just because Jenkins timed out.  I think Mike bumped the timeout for the build that's happening right now.